tests/HtmlMatcherTest.php

```diff
@@ -78,14 +78,12 @@ public function considersValidHtml_WHtmlContainsScriptTagWithHtmlContents()
     public function addsSpecificTextInsideTheSciptTagsInsteadOfItsContents()
     {
         $html = "<div>
-<script type='x-template'>
-	<span></span>
-</script>
+<script type='x-template'><span></span></script>
 </div>";
 
         assertThat($html, is(htmlPiece(havingChild(
             both(withTagName('script'))
-                ->andAlso(havingTextContents(HtmlMatcher::SCRIPT_BODY_REPLACEMENT))))));
+                ->andAlso(havingTextContents("<span><\\/span>"))))));
```

Comment:
> I'm afraid I don't get the problem. Two of the three examples you gave are allowed when using the MediaWiki code style, including the one you refer to as "most consistent".

With middle style it is hard to track arguments position. Last one takes too much vertical space for this simple task.

>  Significant control structures like a return … should always be on their own line.

1. Quiet bold statement. Why? In many projects `if ( … ) return …;` structure considered fine.
2. Then what about [PHP RFC: Arrow Functions](https://wiki.php.net/rfc/arrow_functions) which introduces `fn($x) => $arr[$x]` syntax with implicit `return`? I suppose those should not be allowed at all then?

> Sorry, but I don't understand which one is causing problems and why.

I'm talking about consistency in indentation. We can have a strict rule that each closing bracket goes on a new line, but then we end up with expressions using too much of a vertical space which doesn't bring any readability, I think it even makes it worse.